### ISSUE TITLE
Accessibility: WCAG 2.1 - phase 2+

### ISSUE THREAD

**Issue by SimoneZaza** [cherrytwist/Coordination#1152]
## Description

People who can't use a mouse or trackpad, often (if not always) navigate by using their keyboard. You can experience this yourself by using the tab-key and your arrow-keys. You might notice that although the order on most pages of the platform is correct, you often can't actually see where you are. Some browsers indicate this better than other browsers do, but there are many buttons/links that do not have any indication that they're ''highlighted''.
Moreover, you might also notice that you can't access some of the content, for example the info icons and some of the tooltips. 

## Initiative / goal

When using the keyboard, it should be clear where you are on the page. Would be great if we can also style it a bit, so it aligns nicely with the rest of the content (while of course still standing out to be clearly visible)

### Hypothesis

Adding indications of where you are on the page will improve the accessibility for keyboard users.

## Acceptance criteria and must have scope
Results from testing KN, some issues are still open:  v1: https://alkem.io/building-alkemio-org/challenges/testingexperiment-3692/opportunities/testscenarios-8213/collaboration/testresults-7379/posts/testresults-skip-909 

Issues discovered while testing are listed in this post: https://alkem.io/building-alkemio-org/challenges/testingexperiment-3692/opportunities/testscenarios-8213/collaboration/testresults-7379/posts/test-labelsonall-4674

- [ ] Analysis on what content can't be accessed
- [ ] Analysis on the order of the tabbing, if this aligns with the visual order
- [ ] Fixing findings
- [ ] All items to have an indication

## Stakeholders

UWV, Envision

## Timeline

What's the timeline for this Epic, what resources are needed, and what might potentially block this from hitting the projected end date.

## Design

Figma file:
_Comments are highly appreciated. When commenting in Figma, please also leave a note in the comments box below this issue to notify us_
